.github/workflows/pre-release.yml

```diff
@@ -104,6 +104,23 @@ jobs:
           exit 1
           fi
 
+      # Upgrade TS gRPC code snippets if new version is provided
+      - name: Upgrade TS Restate SDK for gRPC code snippets
+        if: github.event.inputs.sdkTypescriptVersion != ''
+        run: npm --prefix code_snippets/ts-grpc install @restatedev/restate-sdk@^${{ inputs.sdkTypescriptVersion }}
+
+      # Test if TS code snippets compile and build
+      - name: Compile TypeScript gRPC code snippets
+        run: npm install --prefix code_snippets/ts-grpc && npm run proto --prefix code_snippets/ts-grpc && npm run build --prefix code_snippets/ts-grpc
+      - name: Check if TypeScript gRPC code snippets build is successful
+        run: |
+          if [ -d "code_snippets/ts-grpc/dist" ]; then
+          echo "TypeScript gRPC code snippets compilation successful"
+          else
+          echo "TypeScript gRPC code snippets compilation failed"
+          exit 1
+          fi
+
```

Comment:
I guess this means that npm run build --prefix code_snippets/ts-grpc won't exit with a non-zero exit code if the compilation fails, right?

Reply:
I can probably remove this step indeed. The build itself should just fail